addons/deSEC/onlyoffice_docker_desec.sh

```diff
@@ -17,7 +17,7 @@ debug_mode
 # Check if root
```

Comment:
Why did you move the script? Shouldnt be there still a way to runn it without desec?

Reply:
It's not ready yet, but thanks for pointing that out. Actually missed that one.